src/fairchem/core/units/mlip_unit/mlip_unit.py

```diff
@@ -1017,18 +1055,20 @@ def predict(
             tf32_context_manager() if self.inference_mode.tf32 else nullcontext()
         )
 
-        pred_output = {}
-        with inference_context, tf32_context:
-            output = self.model(data_device)
-            for task_name, task in self.tasks.items():
-                pred_output[task_name] = task.normalizer.denorm(
-                    output[task_name][task.property]
-                )
-                if undo_element_references and task.element_references is not None:
-                    pred_output[task_name] = task.element_references.undo_refs(
-                        data_device, pred_output[task_name]
+        if len(data.atomic_numbers) == 1:
```

Comment:
We should add a warning/info message here that says these are coming from DFT and not model predictions. Just for full transparency